queries/java/injections.scm

```diff
@@ -1 +1,6 @@
 [(block_comment) (line_comment)] @comment
+
+(method_invocation
+  name: (identifier) @_method
+  (#eq? @_method "format")
+  arguments: (argument_list . (string_literal) @printf))
```

Comment:
Java has `System.out.format` and `System.out.printf`, which are also printf format strings. Can they be supported as well?